services/form.service.js

```diff
@@ -0,0 +1,87 @@
+const FormModel = require('../models/form.schema');
+
+const getFormByType = async (formType) => {
```

Comment:
🚫 **[eslint]** <[consistent-return](https://eslint.org/docs/rules/consistent-return)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected to return a value at the end of async arrow function.